src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1021,5 +1048,117 @@ public void Method()
 ";
             await VerifyItemExistsAsync(markup, "nullables");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async void CustomNamingStyleInsideClass()
+        {
+            var workspace = WorkspaceFixture.GetWorkspace();
+            var originalOptions = workspace.Options;
+
+            try
+            {
+                workspace.Options = workspace.Options.WithChangedOption(
+                    new OptionKey(SimplificationOptions.NamingPreferences, LanguageNames.CSharp),
+                    NamesEndWithSuffixPreferences());
+
+                var markup = @"
+class Configuration
+{
+    Configuration $$
+}
+";
+                await VerifyItemExistsAsync(markup, "ConfigurationField", glyph: (int)Glyph.FieldPublic,
+                    expectedDescriptionOrNull: CSharpFeaturesResources.Suggested_name);
+                await VerifyItemExistsAsync(markup, "ConfigurationProperty", glyph: (int)Glyph.PropertyPublic,
+                    expectedDescriptionOrNull: CSharpFeaturesResources.Suggested_name);
+                await VerifyItemExistsAsync(markup, "ConfigurationMethod", glyph: (int)Glyph.MethodPublic,
+                    expectedDescriptionOrNull: CSharpFeaturesResources.Suggested_name);
+                await VerifyItemIsAbsentAsync(markup, "ConfigurationLocalFunction");
+            }
+            finally
+            {
+                workspace.Options = originalOptions;
+            }
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async void CustomNamingStyleInsideMethod()
```

Comment:
:exclamation: do not use async void

Reply:
thanks, my mistake

Reply:
It looks like all the other tests in this file use async void as well (and i just copied it), should i fix them as well?

Reply:
You only need to fix the ones you added. 🙂

Reply:
fixed in 8d0ad732daf2ff398ee11e42155a7ba0cac49d5f